src/lib/structures/expr.ml

```diff
@@ -427,36 +427,65 @@ let rec print_silent fmt t =
     begin
       match lit, xs with
       | Sy.L_eq, a::l ->
-        fprintf fmt "(%a%a)"
-          print a (fun fmt -> List.iter (fprintf fmt " = %a" print)) l
+        if get_output_smtlib () then
```

Comment:
Rather than matching for `get_output_smtlib ()` in each case, it'd be nicer to have two functions, say `print_silent_smtlib` and `print_silent_ae`, and then have
```
let print_silent ... = if get_output_smtlib () then print_silent_smtlib ... else print_silent_ae ...
```